src/drivers/gpio/stm32/stm32_gpio_conf_l0x0.h

```diff
@@ -0,0 +1,86 @@
+/**
+ * @file
+ *
+ * @date 07.02.2021
+ * @author Vadim Deryabkin
+ */
+
+#ifndef SRC_DRIVERS_GPIO_STM32_STM32_GPIO_CONF_L0X0_H_
+#define SRC_DRIVERS_GPIO_STM32_STM32_GPIO_CONF_L0X0_H_
+
+#include <hal/reg.h>
+
+#define PINS_NUMBER 16
+#define STM32_GPIO_PORTS_COUNT 6
+
+#define REG_RCC        0x40021000       // Doc: DocID031151 Rev 1, RM0451, 39/774.
```

Comment:
Maybe. would it be better if it was extracted as an option in Mybuild?

Reply:
That are constant for l0x0 series. For other l0 series struct has different data.

Reply:
Yes, I see
But I mean You can specify this parameter in mods.conf not directly in code